### PR TITLE
add waywiser tech note

### PR DESCRIPTION
**Pull request by maelle**
Add post by @mikemahoney218 :pray: (I only added a link to tidymodels, to the software review page of the website, and reformatted footnotes)

* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name. (@mikemahoney218 I added a link to your Mastodon profile)
* [x] I have added relevant tags after browsing existing tags (including "community" tag).
* [x] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.
* [x] I have added the tags - Software Peer Review, my-packagename.
* [x] I have added the package-version YAML tag.
* [x] I have added acknowledgement of the reviewers' work (with links to reviewers).
* [x] I have added a link to the software peer review thread.

### CODE REVIEW

File: content/blog/2023-04-04-waywiser-is-now-a-part-of-ropensci/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+slug: "post-template"
+title: waywiser is Now a Part of rOpenSci
+author:
+  - Mike Mahoney
+# Set the date below to the publication date of your post
+date: 2023-04-04
```
Comment (maelle) [inline content/blog/2023-04-04-waywiser-is-now-a-part-of-ropensci/index.md:7]: Mostly a placeholder!

File: content/blog/2023-04-04-waywiser-is-now-a-part-of-ropensci/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+I'm thrilled to share that [waywiser](https://docs.ropensci.org/waywiser/), my R package focused on providing framework-agnostic (but [tidymodels](https://www.tidymodels.org/)-friendly) methods for assessing models fit to spatial data[^terminology], has passed [peer review](/software-review/) and been accepted to [rOpenSci](/). As always, [the review process](https://github.com/ropensci/software-review/issues/571) improved the package immensely, thanks to the thoughtful reviews of [Virgilio Gómez-Rubio](https://becarioprecario.github.io/) and [Jakub Nowosad](https://jakubnowosad.com/)[^book], as well as the shepherding of [Anna Krystalli](https://www.r-rse.eu/) and [Paula Moraga](https://www.paulamoraga.com/) as editors.
+
+As of Monday, March the 20th, the reviewed version has officially [made its way to CRAN](https://cran.r-project.org/package=waywiser). This is a [huge update](https://docs.ropensci.org/waywiser/news/index.html#waywiser-030), bringing in a ton of new functions and improving consistency and speed across the package, and I'm excited to have it officially released.
```
Comment (maelle) [inline content/blog/2023-04-04-waywiser-is-now-a-part-of-ropensci/index.md:27]: I added which Monday it is

**Comment by mikemahoney218** [comment]
All looks and sounds great to me -- thanks @maelle for doing all the work here :smile:

For tweet and toot, what about something like:
```
[technote] waywiser is Now a Part of rOpenSci 🗺️  🎉

Assessing models fit to spatial data is hard! The waywiser package makes it a bit easier. @MikeMahoney218@fosstodon.org celebrates the addition of waywiser to rOpenSci, and shares a preprint diving into the details of the package.

https://ropensci.org/blog/2023/04/04/waywiser-is-now-a-part-of-ropensci/

#RStats

Cross-posted from Mike's personal blog: https://www.mm218.dev/posts/2023-03-21-waywiser-ropensci/

🙏 And of course, big thanks to rOpenSci reviewers @becarioprecario@mathstodon.xyz, @nowosad@fosstodon.org
```

Don't know how to condense that into a tweet, but feel free to drop the link to my blog -- perfectly happy to just have it linked from the rOpenSci post itself :smile:

**Comment by steffilazerte** [comment]
Thanks @mikemahoney218! I'll just split it in two, I do that often enough 😁

**Pull request merged**